docs/contents/components/forms/dropzone/index.ja.mdx

```diff
@@ -128,10 +116,24 @@ import {
 ```tsx
 <Dropzone maxSize={3 * 1024 ** 2}>
   <VStack w="auto" gap="2xs">
-    <Text fontSize="xl">Drag file here or click to select file</Text>
-    <Text fontSize="sm">
-      Attach as many files as you like, each file should not exceed 3mb
-    </Text>
+    <For
```

Comment:
There is no need to use `For` here. I feel that the modification makes it less readable.